src/main/java/juja/sqlcmd/view/Console.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Scanner;
 
 public class Console implements View {
-    Scanner scanner;
+    private Scanner scanner;
```

Comment:
Why to have a Scanner as a field? You don't return it anywhere and will never reuse somewhere else... I think it's redundant